examples/ConfigStoreDemo/Program.cs

```diff
@@ -24,9 +26,13 @@ public static IWebHost BuildWebHost(string[] args)
                     // 3. Set up the provider to listen for changes to the background color key-value in Azure App Configuration
 
                     var settings = config.AddJsonFile("appsettings.json").Build();
+
+                    IConfigurationSection endpointsSection = settings.GetSection("endpoints");
+                    var endpoints = endpointsSection.GetChildren().AsEnumerable().Select(endpoint => new Uri(endpoint.Value));
```

Comment:
I'm not sure that we should be complicating our base example. New users most likely will want the simplest setup possible when they go looking for an example.

Reply:
I agree. I don't expect the basic sample  is updated.

Reply:
Let's leave the web sample alone and only update he console app sample.